app/models/pieces/pawn.rb

```diff
@@ -1,2 +1,13 @@
 class Pawn < Piece
+  def valid_move?(x_destination, y_destination)
+    x_dist = (x_coord - x_destination).abs
+    y_dist = (y_coord - y_destination).abs
+
+    if y_coord == 1 || 6
+    	y_dist == 2
```

Comment:
This only allows the first move to move with a y delta of 2.  y_dist can be 1 or 2.  I think you want:

`y_dist <= 2`

Instead.